lib/Optimizer/GraphOptimizer/NodeSplitting.cpp

```diff
@@ -1515,8 +1419,8 @@ glow::splitNodes(Function *F, const SplitNodeOptionMap &splitOptionMap,
 
   // Since we will be transforming the original list of nodes, reverse iterate.
   auto &nodes = F->getNodes();
-  for (auto it = nodes.rbegin(), e = nodes.rend(); it != e; it++) {
-    Node *node = &*it;
+  for (auto it = nodes.rbegin(), e = nodes.rend(); it != e;) {
+    Node *node = &*(it++);
```

Comment:
No big deal, but curious why you made this change.

Reply:
If I remember correctly this is because this PR erases the original node automatically upon splitting so when doing this in a loop the current iterator is invalided.